gossip/src/crds.rs

```diff
@@ -235,6 +237,9 @@ impl Crds {
         let label = value.label();
         let pubkey = value.pubkey();
         let value = VersionedCrdsValue::new(value, self.cursor, now, route);
+        if let GossipRoute::PushMessage(_) = route {
+            self.stats.lock().unwrap().num_total_push += 1
+        }
```

Comment:
This might be a duplicate of `packets_received_push_messages_count`:
https://github.com/anza-xyz/agave/blob/a916edb7a/gossip/src/cluster_info.rs#L2558-L2560

Reply:
So `packets_received_push_messages_count` counts the actual messages received. but `num_total_push` counts the actual `CrdsValue`s, since more than one `CrdsValue` can be in a message. See: https://github.com/anza-xyz/agave/blob/a916edb7a2caaa01eda171d67a1d8f1dc1f397b1/gossip/src/crds_gossip_push.rs#L140-L147.

Reply:
Oh, good point.

how about this two metrics?
https://github.com/anza-xyz/agave/blob/80d3200f4/gossip/src/cluster_info_metrics.rs#L647-L651
https://github.com/anza-xyz/agave/blob/80d3200f4/gossip/src/cluster_info_metrics.rs#L696

isn't their sum equal to the `num_total_push`?